### PR TITLE
fix: playwright bump to v1.37.0

### PR DESCRIPTION
**Pull request by nullxDEADBEEF**
# Description

Please include a summary of the change and which issue is fixed.

Fixes #

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -17,4 +17,5 @@ jobs:
           PUBLIC_ENV=test"
           echo "$variables" > .env
           yarn install
+          npx playwright install
```
Comment (AndreasPB) [inline .github/workflows/e2e.yml:20]: Why? :D

Comment (nullxDEADBEEF) [reply .github/workflows/e2e.yml:20]: This is to make sure we have new browsers, the CI failed with this before: 
Screenshot 2023-08-20 at 14 46 59

Comment (AndreasPB) [reply .github/workflows/e2e.yml:20]: Ahh okay

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**